site/templates/application/footer/social-media-icons.jet

```diff
@@ -0,0 +1,99 @@
+{{ block footerSocialMediaIcons() }}
+  {{ social_link_instagram := config("social_link_instagram") }}
+  {{ social_link_youtube := config("social_link_youtube") }}
+  {{ social_link_facebook := config("social_link_facebook") }}
+  {{ social_link_twitter := config("social_link_twitter") }}
+  {{ social_link_letterboxd := config("social_link_letterboxd") }}
+  {{ social_link_linkedin := config("social_link_linkedin") }}
+
+  {{ if social_link_instagram != "" || social_link_youtube != "" || social_link_facebook != "" || social_link_twitter != "" || social_link_letterboxd != "" || social_link_linkedin != "" }}
+    <div class="footer-social-media-icons">
+      {{ yield footerSocialMediaIcon(link=social_link_instagram, iconClass="instagram", titleKey="wcag_aria_label_social_instagram") }}
+      {{ yield footerSocialMediaIcon(link=social_link_youtube, iconClass="youtube", titleKey="wcag_aria_label_social_youtube") }}
+      {{ yield footerSocialMediaIcon(link=social_link_facebook, iconClass="facebook", titleKey="wcag_aria_label_social_facebook") }}
+      {{ yield footerSocialMediaIcon(link=social_link_twitter, iconClass="twitter", titleKey="wcag_aria_label_social_twitter") }}
+      {{ yield footerSocialMediaIcon(link=social_link_letterboxd, iconClass="letterboxd", titleKey="wcag_aria_label_social_letterboxd") }}
+      {{ yield footerSocialMediaIcon(link=social_link_linkedin, iconClass="linkedin", titleKey="wcag_aria_label_social_linkedin") }}
+    </div>
+  {{ end }}
+{{ end }}
+
+{{ block footerSocialMediaIcon(link, iconClass, titleKey) }}
+  {{ if link != "" }}
+    <a class="s72-btn" target="_blank" title="{{i18n(titleKey)}}" href="{{link}}">
+      {{ if iconClass == "instagram" }}
+        {{ yield instagramIcon() }}
+      {{ end }}
+      {{ if iconClass == "youtube" }}
+        {{ yield youtubeIcon() }}
+      {{ end }}
+      {{ if iconClass == "facebook" }}
+        {{ yield facebookIcon() }}
+      {{ end }}
+      {{ if iconClass == "twitter" }}
+        {{ yield twitterIcon() }}
+      {{ end }}
+      {{ if iconClass == "letterboxd" }}
+        {{ yield letterboxdIcon() }}
+      {{ end }}      
+      {{ if iconClass == "linkedin" }}
+        {{ yield linkedinIcon() }}
+      {{ end }}                      
+    </a>
+  {{ end }}
+{{ end }}
+
+{{ block instagramIcon() }}
+<div class="social-link-button instagram">
```

Comment:
The contents of these blocks all need indenting.

Reply:
They can also all be simplified by hoisting `social-link-button` div up into the `footerSocialMediaIcon` block.